addons/l10n_id_efaktur_coretax/models/uom_code.py

```diff
@@ -0,0 +1,11 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+class EfakturUomCode(models.Model):
+    _name = "l10n_id_efaktur_coretax.uom.code"
+    _description = "UOM categorization according to E-Faktur"
+
+    code = fields.Char()
+    uom_name = fields.Char()
```

Comment:
You don't define a new for this one? How would it look in the view?

Reply:
you mean `name` for this one?

Reply:
good point,  I'll use `name` instead of `uom_name` here in this case.

Reply:
noted